internal/model/live_update.go

```diff
@@ -5,16 +5,31 @@ import (
 )
 
 // Specifies how to update a running container.
+// 0. If any paths specified in a FallBackOn step have changed, fall back to an image build
+//    (i.e. don't do a LiveUpdate)
 // 1. If there are Sync steps in `Steps`, files will be synced as specified.
 // 2. Any time we sync one or more files, all Run and RestartContainer steps will be evaluated.
 type LiveUpdate struct {
-	Steps []LiveUpdateStep
-
-	// When files matching any of these paths change, we should fall back to a full rebuild.
-	FullRebuildTriggers PathSet
+	Steps   []LiveUpdateStep
+	BaseDir string // directory where the LiveUpdate was initialized (we'll use this to eval. any relative paths)
 }
 
-func NewLiveUpdate(steps []LiveUpdateStep, fullRebuildTriggers PathSet) (LiveUpdate, error) {
+func NewLiveUpdate(steps []LiveUpdateStep, baseDir string) (LiveUpdate, error) {
+	// Check that all FallBackOn steps come at the beginning
+	// (Technically could do this in the loop below, but it's
+	// easier to reason about/modify this way.)
+	seenNonFallBackStep := false
+	for _, step := range steps {
+		switch step.(type) {
+		case LiveUpdateFallBackOnStep:
```

Comment:
is it possible to have more than one fallback step? if not, do you want to assert that?

Reply:
yep, it IS possible to have more than one!